doc/examples/Finite_State_Projection/utils.py

```diff
@@ -12,9 +13,7 @@ def plot_AMICI(sbml_dir: str,
                t: np.ndarray,
                r_max: int,
                p_max: int):
-    """
-    Compiles, simulates and plots the AMICI model for the FSP example.
-    """
+    """Compiles, simulates and plots the AMICI model for the FSP example."""
```

Comment:
```suggestion
    """Compile, simulate and plot the AMICI model for the FSP example."""
```
active/imperative usually preferable